ngrams_across_time/utils/data.py

```diff
@@ -0,0 +1,27 @@
+from torch.utils.data import Dataset as TorchDataset
+from typing import List
+
+
+
+class MultiOrderDataset(TorchDataset):
```

Comment:
This could be called ZippedDataset or something to make it clear it's not doing anything order-specific